### ISSUE TITLE
Google Local Results Missing from JSON

### ISSUE THREAD

**Issue by beingskyler** [serpapi/public-roadmap#130]
Ran a test on approx. 10,000 queries. SERPAPI failed to return JSON data for `local_results` on 99% of queries.

## Example Failure:
- Query: dallas truck accident attorneys
- Playground String: https://serpapi.com/playground?q=dallas+truck+accident+attorneys&location=United+States&gl=us&hl=en

## Example Success:
- Query: henderson truck accident lawyers
- Playground String: https://serpapi.com/playground?q=henderson+truck+accident+lawyers&location=United+States&gl=us&hl=en

## Possible Issues
I've noticed that whenever `local_results` appear below another element (e.g. `local_ads` or `organic_results`) things fail. However, I found several other instances where the JSON failed to capture `local_results` despite them being the first element on the SERP as well.

**Comment by tanys123** [comment]
Hey @andywaite, we have PR ready for this, waiting for the team to review and deploy.

**Comment by andywaite** [comment]
@tanys123 that's awesome news.

Apologies - I'm not familiar with your process for getting this into your live environment. How will we know when this is live?

**Comment by tanys123** [comment]
Hi @andywaite, once PR is merged, it will be deployed in few minutes. As for now, the changes already deployed, you can try again and set `no_cache=true`. Let me know if the issue persist.

**Comment by andywaite** [comment]
@tanys123 we're still seeing issues sadly

I used the search on your homepage to try a few different terms I'd expect to give local results. The page on the left appears to show a local pack with a map & 3 GBP results. However the JSON body of the response doesn't contain a structured version of these results.

<img width="1204" alt="Screenshot 2022-06-19 at 18 53 14" src="https://user-images.githubusercontent.com/6773151/174494057-60f9328c-6ece-4193-9ce2-587f62c07f2a.png">

**Comment by tanys123** [comment]
@andywaite Thanks for verifying it. Google has updated their HTML and I have created a PR to address it.

**Comment by hartator** [comment]
I think I created a duplicate of this: https://github.com/serpapi/public-roadmap/issues/184. Feel free to close if not helpful. 👍
